pyCardDeck/standard_deck.yml

```diff
@@ -0,0 +1,61 @@
+!!python/object:pyCardDeck.deck.Deck
+_cards:
+- !!python/object:pyCardDeck.cards.PokerCard {name: Seven of Spades, rank: '7', suit: Spades}
```

Comment:
It will always be constructed from cards.PokerCard

Since it completely rewrites the object it will not matter what the original cards were constructed from